sporkfish/engine.py

```diff
@@ -39,8 +46,32 @@ def best_move(self, board: Board, timeout: Optional[float] = None) -> chess.Move
         :return: The selected chess move.
         :rtype: chess.Move
         """
-        opening_move = self._opening_book.query(board)
-        return opening_move or self._searcher.search(board, timeout)[1]
+        if opening_move := self._opening_book.query(board):
+            return opening_move
+
+        end_move = self._use_endgame_tablebase(board)
+        return end_move or self._searcher.search(board, timeout)[1]
+
+    def _use_endgame_tablebase(self, board: Board) -> Optional[chess.Move]:
+        """
+        Return endgame tablebase move if applicable.
+
+        :param board: The current chess board position.
+        :type board: Board
+        :return: The selected chess move.
+        :rtype: Optional[chess.Move]
+        """
+        counter = 0
+        for square in chess.SQUARES:
+            # Search board and count number of pieces present- if less than threshold trigger endgame tablebase.
+            if board.piece_at(square):
+                counter += 1
+            if counter > 6:
+                break
+        if counter <= 6:
```

Comment:
I suspect we don't have TBs worth of space. Btw possibly in future we will query the table bases from lichess using lila - but this will be a good in between method...

Reply:
#104.

Reply:
Don't know what lila is - but sounds like you have a plan in mind - good good.

Reply:
Sorry, meant this: https://github.com/lichess-org/lila-tablebase couldnt type on phone. Will add to issue.